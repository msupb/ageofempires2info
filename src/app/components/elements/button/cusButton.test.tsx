import React from 'react';
import ReactDom from 'react-dom';
import CusButton from './cusButton';

const onTestClick = () => {
    return "Test click";
}

test("CusButton will render without crashing", () => {
    const div = document.createElement('div');
    ReactDom.render(<CusButton btnText="Test" btnType="button" onClickMethod={onTestClick}></CusButton>, div)
})