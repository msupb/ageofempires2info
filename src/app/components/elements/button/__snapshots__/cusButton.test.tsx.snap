// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<button
  className="cusButton"
  data-testid="cusButtonTest"
  onClick={[Function]}
  type="button"
>
  Test
</button>
`;
